sale_order_amount_to_invoice/views/sale_order_view.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <record id="view_order_form" model="ir.ui.view">
+        <field name="name">sale.order.form</field>
+        <field name="model">sale.order</field>
+        <field name="inherit_id" ref="sale.view_order_form" />
+        <field name="priority">99</field>
+        <field name="arch" type="xml">
+            <xpath
+                expr="//field[@name='order_line']/tree//field[@name='company_id']"
+                position="after"
+            >
+                <field
+                    name="untaxed_amount_to_invoice"
+                    string="To Invoice"
+                    optional="hide"
+                />
+            </xpath>
+            <xpath expr="//field[@name='price_tax']" position="after">
```

Comment:
To maintain original behavior, it would be good put this field after "amount_total" IMO.

Reply:
Just fixed, let me know if I got something wrong. Thank you for reviewing the code!